WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsV4PeriodViewController.swift

```diff
@@ -685,15 +754,24 @@ private extension StoreStatsV4PeriodViewController {
     }
 
     func reloadSiteFields() {
-        guard visitorsData != nil else {
-            return
-        }
+        // TODO-5450: Temporary solution to make the empty view show up.
+        // Remove this when JCP site is listed and can be selected.
+        if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.jetpackConnectionPackageSupport) {
```

Comment:
I wonder if we could make the view controller support all three `SiteVisitStatsMode` cases, and only show `redactedDueToJetpack` case when the feature flag is enabled? This way, we only need one feature flag check 😄

Reply:
That makes sense! I've updated this in 7784b1283f49d0577706714f5d64c1ba0a33a3a4.